config/locales/sp.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
You need to rename the file to `es.json`, or it will not work.

Reply:
Just updated filename